bin/client

```diff
@@ -43,7 +46,11 @@ lt_client(opt.port, opt, function(err, tunnel) {
     }
 
     console.log('your url is: %s', tunnel.url);
-
+    
+    if (argv.open) {
+      require('openurl').open(tunnel.url);
```

Comment:
Sorry, I didn't see I was missing 2 blank spaces. I also move the require('openurl') to the top of the file.